Documentation/Wiki/How-To-Run-BuildXL/Resource-Usage-Configuration.md

```diff
@@ -26,21 +27,33 @@ Some pips are primitives for IO operations. These are:
 
     /maxIO:<number of concurrent I/O operations>
                                 Specifies the maximum number of I/O operations that BuildXL will launch at one time. The
-                                default value is 1/4 of the number of processors in the current machine, but at least
-                                1.
+                                default value is 1/4 of the number of processors in the current machine, but at least 1.
     /maxIOMultiplier:<double>   
-                                Specifies maxIO in terms of a multiplier of the machine's processor count. The default
-                                is 0.25.
+                                Specifies maxIO in terms of a multiplier of the machine's processor count. The default is 0.25.
+
+## Cache Concurrency limits
+Since cache operations use different resources (IO, CPU, network) concurrently, they have a different concurrency limit settings.
+
+    /maxCacheLookup:<number of concurrent operations>
+                                Specifies the maximum number of cache lookup operations that BuildXL will launch at one time. 
+                                The default value is 2 times the number of processors in the current machine.
+    /maxMaterialize:<number of concurrent operations>   
+                                Specifies the maximum number of concurrent materialize operations (e.g., materialize inputs, storing two-phase cache entries, analyzing pip violations). 
+                                The default value is 2 times the number of processors in the current machine.
 
-## Low priority
-Processes in Windows use the "Normal" priority by default but can be overridden with another setting. This can be manipulated in Task Manager but BuildXL accepts it as a command line option as well. Generally this is less effective than setting the concurrency limit directly.
+## Cache Concurrency limits
+Since cache operations use different resources (IO, CPU, network) concurrently, they have a different concurrency limit settings.
 
-    /lowPriority[+|-]           Runs the build engine and all tools at a lower priority in order to provide better
-                                responsiveness to interactive processes on the current machine.
+    /maxCacheLookup:<number of concurrent operations>
+                                Specifies the maximum number of cache lookup operations that BuildXL will launch at one time. 
+                                The default value is 2 times the number of processors in the current machine.
+    /maxMaterialize:<number of concurrent operations>   
+                                Specifies the maximum number of concurrent materialize operations (e.g., materialize inputs, storing two-phase cache entries, analyzing pip violations). 
+                                The default value is 2 times the number of processors in the current machine.
 
 ## Memory utilization ##

```

Comment:
> Memory utilization  [](start = 2, length = 20)

Is it worth adding a note that the default values are very conservative which in essence make this feature disabled by default?